src/auditwheel/policy/__init__.py

```diff
@@ -3,21 +3,107 @@
 import json
 import logging
 import platform as _platform_module
+import re
 import sys
 from collections import defaultdict
 from os.path import abspath, dirname, join
 from pathlib import Path
+from typing import Any, Generator
+
+from auditwheel.elfutils import filter_undefined_symbols, is_subdir
 
 from ..libc import Libc, get_libc
 from ..musllinux import find_musl_libc, get_musl_version
 
 _HERE = Path(__file__).parent
+LIBPYTHON_RE = re.compile(r"^libpython\d+\.\d+m?.so(.\d)*$")
 
 logger = logging.getLogger(__name__)
 
 # https://docs.python.org/3/library/platform.html#platform.architecture
 bits = 8 * (8 if sys.maxsize > 2**32 else 4)
 
+_POLICY_JSON_MAP = {
+    Libc.GLIBC: _HERE / "manylinux-policy.json",
+    Libc.MUSL: _HERE / "musllinux-policy.json",
+}
+
+
+class WheelPolicies:
+    @staticmethod
+    def load():
+        libc_variant = get_libc()
+        policies_path = _POLICY_JSON_MAP[libc_variant]
+        policy_dict = json.loads(policies_path.read_text())
+        return WheelPolicies(policy_dict)
+
+    def __init__(self, policies: dict) -> None:
```

Comment:
c.f. previous comment on `load()`

Maybe it would also be good to get arch/libc/musl as optional arguments in `__init__` but that could come later (I'm thinking about cross-platform repair / cross-platform check).

Reply:
> Maybe it would also be good to get arch/libc/musl as optional arguments in `__init__` but that could come later (I'm thinking about cross-platform repair / cross-platform check).

This would be great. My project does cross-platform repair and jumps through module loading [hoops](https://github.com/jvolkman/repairwheel/blob/main/src/repairwheel/linux/monkeypatch.py#L6) currently. Although having everything in a `WheelPolicies` class should already help with that patching.

Reply:
@mayeut Do you mind if we do this in a follow up?
@jvolkman If you do end up needing this functionality, would you be willing to make a PR for it?

Reply:
> Do you mind if we do this in a follow up?

I don't mind